compiler-core/src/ast/untyped.rs

```diff
@@ -293,11 +294,18 @@ impl HasLocation for UntypedExpr {
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub enum FunctionLiteralKind {
     Capture { hole: SrcSpan },
-    Anonymous { head: SrcSpan },
+    Anonymous { head: SrcSpan, body: SrcSpan },
```

Comment:
Isn't the start of `body` always the end of `head`?

This data also is duplicating the data of the containing function data structure I think. It would be fab to not have that duplication there